src/main/java/frc/robot/subsystems/intake/IntakeIO.kt

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems.intake
+
+import edu.wpi.first.units.Current
+import edu.wpi.first.units.Distance
+import edu.wpi.first.units.Measure
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Velocity
+import edu.wpi.first.units.Voltage
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+
+interface IntakeIO {
+    class IntakeInputs : LoggableInputs {
+        var intakeLinearVelocity: MutableMeasure<Velocity<Distance>> = MutableMeasure.zero(Units.MetersPerSecond)
+        var intakeSupplyVoltage: MutableMeasure<Voltage> = MutableMeasure.zero(Units.Volts)
+        var intakeMotorVoltage: MutableMeasure<Voltage> = MutableMeasure.zero(Units.Volts)
+        var intakeStatorCurrent: MutableMeasure<Current> = MutableMeasure.zero(Units.Amps)
```

Comment:
Sorry for not mentioning this the first time, but the "intake" prefixing all of these variables is redundant. Please remove to match conventions of other io layers.